quorum/templates/08-epirus-chain-explorer.yaml

```diff
@@ -0,0 +1,61 @@
+{{ $scope := . }}
+{{- if eq $scope.Values.epirus.enabled true }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: epirus-chain-explorer-deployment
+spec:
+  strategy:
+    type: Recreate
+  replicas: 1
+  selector:
+    matchLabels:
+      name: epirus-chain-explorer-deployment
+  template:
+    metadata:
+      name:  epirus-chain-explorer-deployment
+      labels:
+        app: epirus
+        tier: backend
+        name: epirus-chain-explorer-deployment
+    spec:
+      securityContext:
+      containers:
+      - name: epirus-api
+        image: web3labs/epirus-free-api:latest
+        ports:
+          - containerPort: 8090
+        env:
+          - name: NODE_ENDPOINT
+            value: http://{{$scope.Values.epirus.node}}.quorum-network.svc.cluster.local:8545
+          - name: MONGO_CLIENT_URI
+            value: mongodb://localhost:27017
+          - name: REINDEX_ENDPOINT
```

Comment:
Is this really used and does it work? I do not see where the `ingestion` host is defined

Reply:
removed this env variable as it is not used